e2e-tests/README.md

```diff
@@ -109,12 +110,9 @@ Output of a test run should look something like this:
 
     ============== 63 passed, 10 xfailed in 970.27 seconds ===============
 
-__Note__
-"~" will not resolve to the home directory when used in the py.test command line.
```

Comment:
I tested this and it is not true, at least for test paths. If anyone is able to replicate this please file a bug and mention me. We should fix it.